cmd/restic/cmd_check.go

```diff
@@ -308,9 +354,9 @@ func runCheck(opts CheckOptions, gopts GlobalOptions, args []string) error {
 	if errorsFound {
 		return errors.Fatal("repository contains errors")
 	}
-
+	
```

Comment:
Please always run `gofmt` for every change! I'd recommend to configure it to run automatically.